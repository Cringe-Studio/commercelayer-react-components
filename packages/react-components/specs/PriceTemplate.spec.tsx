import React from 'react'
import PriceTemplate from '../#components/utils/PriceTemplate'
import renderer from 'react-test-renderer'
import PropTypes from 'prop-types'

test('<PriceTemplate/>', () => {
  expect.assertions(3)
  const component = renderer.create(<PriceTemplate />)
  const tree = component.toJSON()
  const root = component.toTree()
  const proptypes = root.type['propTypes']
  expect(tree).toMatchSnapshot()
  expect(proptypes.formattedAmount).toBe(PropTypes.string)
  expect(proptypes.formattedCompare).toBe(PropTypes.string)
})

test('<PriceTemplate check children format />', () => {
  expect.assertions(3)
  const component = renderer.create(<PriceTemplate />)
  const tree = component.toJSON()
  const root = component.toTree()
  const rendered = root.rendered
  expect(tree).toMatchSnapshot()
  expect(rendered.type).toBe('span')
  expect(root.nodeType).toBe('component')
})
