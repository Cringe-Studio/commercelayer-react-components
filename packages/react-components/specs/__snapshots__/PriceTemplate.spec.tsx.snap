// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PriceTemplate check children format /> 1`] = `
<span>
  
</span>
`;

exports[`<PriceTemplate/> 1`] = `
<span>
  
</span>
`;
